firmware/quadruna/BMS/src/app/states/app_allStates.c

```diff
@@ -54,25 +58,62 @@ bool app_allStates_runOnTick100Hz(void)
     app_heartbeatMonitor_tick();
     app_heartbeatMonitor_broadcastFaults();
 
+    const bool balancing_enabled = true; // app_canRx_Debug_CellBalancingRequest_get();
```

Comment:
I'm assuming you did it for testing and probably forgot to revert to CAN Rx flag.

Reply:
Ah yes good catch thank you